Automattic/woocommerce-payments
# Add WC Blocks webpack-rtl-plugin implementation to the project

#### Changes proposed in this Pull Request
Add WC Blocks `webpack-rtl-plugin` [implementation](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/bin/webpack-rtl-plugin.js) to the project.

This change was prompted by compilation errors encountered with WooPayments in the WooPay E2E environment on MacOS.

#### Testing instructions

* Follow the instructions in https://github.com/Automattic/woocommerce-payments/pull/6897 and ensure there are no regressions in the RTL support. 

-------------------


**Post merge**
